scheduler/test/cook/test/config.clj

```diff
@@ -96,3 +97,40 @@
                                [:config :kubernetes :controller-lock-num-shards]
                                256)]
       (is (thrown? ExceptionInfo (config-settings bad-config))))))
+
+(deftest test-valid-gpu-models-config-settings
```

Comment:
I was thinking here you'd just invoke  (guard-invalid-gpu-config  directly and not go through config-settings. 

Then it goes:
(is (guard-invalid-gpu-config {....}))
(is (thrown-with-msg? ExceptionInfo
                   #"Valid GPU models for pool-regex test-pool is not defined"
                   (guard-invalid-gpu-config {....})))

Reply:
done

Reply:
Nicer. Next question. What do you think of removing the (let ...) and just inline the argument?

Reply:
sure, done

Reply:
sure, done